### PR TITLE
Enable Travis CI

### PR DESCRIPTION
**Pull request by ligurio**

### CODE REVIEW

**Comment by ligurio** [comment]
Passed build - https://travis-ci.org/github/tarantool/mysql-tarantool-replication/builds/735535577

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+    - os: linux
+      dist: focal
+      compiler: clang
+
+before_script:
+    - sudo -E apt update -y
+    - sudo -E apt install -y libboost-dev
+
+script:
+    - . ./build.sh
```
Comment (LeonidVas) [inline .travis.yml:22]: Why `. ./build.sh` not just `./build.sh`

File: .travis.yml [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+
+sudo: required
+
+dist: focal
+
+matrix:
+  include:
+    - os: linux
+      dist: bionic
+      compiler: clang
```
Comment (LeonidVas) [inline .travis.yml:11]: Why choose clang compiler? As I can see, we usually use gcc.
Maybe we can specify a "compiler" on one level with "language"?

**Review by LeonidVas — COMMENTED** [review]
Generally OK. But I have some minor questions / comments:
- I think [this](https://trac.osgeo.org/qgis/ticket/1939) is a good description for the commit message "cmake: replace -ldl with ${CMAKE_DL_LIBS} ".
- I suggest to use the tarantool style - "tarantool-c: bump version"

File: .travis.yml (hunk 1 above)
Comment (ligurio) [reply .travis.yml:22]: For unknown reasons `build.sh` is lack of shebang, so it is not really a shell script, it's just a set of commands in a file. Dot loads a source of a file into the current shell script and executes them. Dot has the same meaning as command `source` in Bash, but `.` is a POSIX standard while `source` is not, i.e., it's a Bashism.

File: .travis.yml (hunk 2 above)
Comment (ligurio) [reply .travis.yml:11]: > Why choose clang compiler?

Sometimes compilation fails with gcc, for example [here](https://travis-ci.org/github/tarantool/mysql-tarantool-replication/jobs/735535579):
```
/home/travis/build/tarantool/mysql-tarantool-replication/lib/libslave/field.h:176:8: error: ‘vector’ in namespace ‘std’ does not name a template type
  176 |   std::vector<std::string> str_values;
      |        ^~~~~~
/home/travis/build/tarantool/mysql-tarantool-replication/lib/libslave/field.h:25:1: note: ‘std::vector’ is defined in header ‘<vector>’; did you forget to ‘#include <vector>’?
   24 | #include <mysql/binary_log_funcs.h>
  +++ |+#include <vector>
   25 | /*
```
It is a known problem and described in tutorial included to Tarantool documentation - https://www.tarantool.io/en/doc/latest/tutorials/libslave/

This time I want just to enable CI in a project so I left clang only.

> As I can see, we usually use gcc.

It's better to use both if there are no serious reasons to do not so.
 
> Maybe we can specify a "compiler" on one level with "language"?

No sense to do it as we specify compiler in a matrix.

**Review by LeonidVas — APPROVED** [review]
LGTM

**Pull request merged**